lib/Controller/LocalAttachmentsController.php

```diff
@@ -40,16 +41,21 @@ class LocalAttachmentsController extends Controller {
 	/** @var string */
 	private $userId;
 
+	/** @var ILogger */
+	private $logger;
```

Comment:
prop is never read

Reply:
yes, I've used it to debug, but now I don't need it anymore. Is it really important to remove it? I thought it would be good to leave it for the next developer who happens to need it.

Reply:
It's dead code, so yeah, it should be removed. Clean code is good code :broom: